components/SearchForm.vue

```diff
@@ -1,11 +1,13 @@
 <template>
 	<div class="search-container">
 		<form id="search-form" action="/search">
-			<label for="search">Search for:</label>
-			<input id="search" type="search" placeholder="Search..." name="s">
-			<button type="submit">
-				Search
-			</button>
+			<input
+				name="s"
+				type="search"
+				placeholder="Search..."
+				title="Enter keywords for a site-wide search"
```

Comment:
I think having both a `title` and an `aria-label` might not be necessary but @lliskovoi might have insight.